RELEASE_NOTES.md

```diff
@@ -1,6 +1,7 @@
 # Upcoming
 
 - feat: The IDE will show verification errors for a method immediately after that method has been verified, instead of after all methods are verified.
+- feat: Dafny is now available on NuGet, installable with `dotnet tool install Dafny`.
```

Comment:
```suggestion
- feat: Dafny is now available on NuGet, installable with `dotnet tool install --global Dafny`.
```

Debatable perhaps, but it seems like the most common use case by far is putting it on your PATH so you can just run `dafny` on the command line.

I would also be more specific about the set of packages we're publishing than just "Dafny" :)